### ISSUE TITLE
Color contrast ratio of keyboard focus present on "C#" control is 1.277:1 which is less than 3:1: A11y_Visual Studio Client_Create new project_Non-text Contrast

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#246460]
### GIT Hub Tags:
#A11yTCS; #A11ySev3; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #DesktopApp; #A11yMAS; #WCAG1.4.3; #ColorContrast; #Win11; #FTP;

### Environment and OS details:
Application name: Visual studio code

### Reproduction Steps:

1. Open Visual studio code
2. Open command palette 
4. Open accessibility insights for windows and check color contrast ratio of "Remote" text.
5. Observe that keyboard focus on any quick pick.

### Actual behavior:
Color contrast ratio of keyboard focus is 1.28:1 which is less than 3:1

### Expected behavior:
Color contrast ratio of keyboard focus should be greater than or equal to 3:1

### User Impact:
Low vision users will not be able to understand where the keyboard focus is present if color contrast ratio will be less than 3:1

### Attachment:

![Image](https://github.com/user-attachments/assets/14938c01-94c4-4f8d-b6fb-7ac9d7bbfd21)

**Comment by kupatkar99** [comment]
Primary bug ID:https://github.com/microsoft/vscode-azurefunctions/issues/4482

**Comment by kkbrooks** [comment]
### Proposed design approach

Add a border around the focus list items using --vscode-focusBorder (#0078D4 in dark theme), which passes 3:1 contrast requirements. Adding the border for focus state is a very common pattern used across the web and other Microsoft tools and should meet people's expectations.

#### Design mock
![Image](https://github.com/user-attachments/assets/002d2def-12db-4da8-84e6-dce807c7b2f6)

#### Color contrast check
![Image](https://github.com/user-attachments/assets/754ae035-4435-40c9-8cc6-99d6070df4d6)

cc @hawkticehurst @jo-oikawa @meganrogge

**Comment by hawkticehurst** [comment]
Just opened a PR with a fix for this issue! https://github.com/microsoft/vscode/pull/249937
